sanic/config.py

```diff
@@ -130,4 +136,7 @@ def load_environment_vars(self, prefix=SANIC_PREFIX):
                     try:
                         self[config_key] = float(v)
                     except ValueError:
-                        self[config_key] = v
+                        if v in ["True", "False"]:
```

Comment:
Should we also add other truthy strings? 

`"true", "yes", "Yes", "Y", etc`

Reply:
The `setup.py` uses `distutils.util.strtobool` to [process some environment variables ](https://sanic.readthedocs.io/en/latest/sanic/getting_started.html#install-sanic). One thing that has bothered me in the past is that we are not consistent in how environment variables are handled throughout.

If setting `SANIC_NO_UVLOOP=true` yields a `True` value, I would expect `SANIC_KEEP_ALIVE=true` to also.

Reply:
There is something strange with tests *'-no-ext' when I add 
```
from distutils.util import strtobool
```
in `config.py`

```
E       assert ('asyncio', 4...f2244870da0>') == ('sanic.root',....0.1:42101/'")
E         At index 0 diff: 'asyncio' != 'sanic.root'
E         Use -v to get the full diff

tests/test_middleware.py:106: AssertionError
```
No matter where I put it - in the head or in the function.
Any idea?

Reply:
I added it to `config.py` on a clean version of the `master` branch and was able to run both `tox -e py36-no-ext` and `tox -e py37-no-ext`.

Now, looking at `test_middleware.py` I see exactly what the issue is because I just dealt with this issue on `test_logo.py`. Why it is doing this for you right now ONLY with that import ... that makes no sense, unless somehow logging is being changed somewhere to `DEBUG`.

Basically what is happening is there is an extra debug that is logged, so `caplog` is one row off of stdout from where it should be. At least that is what I was dealing with (see `test_logo.py` to see how I dealt with it). What is that full output?

Reply:
```
E       assert ('asyncio', 4...f609420dd68>') == ('sanic.root',....0.1:42101/'")
E         At index 0 diff: 'asyncio' != 'sanic.root'
E         Full diff:
E         - ('asyncio',
E         + ('sanic.root',
E         40,
E         +  "Exception occurred while handling uri: 'http://127.0.0.1:42101/'")
E         -  'Unclosed client session\n'
E         -  'client_session: <aiohttp.client.ClientSession object at 0x7f609420dd68>')

tests/test_middleware.py:106: AssertionError
```
And it is only fail when I run all the tests. If I run only `test_middleware.py` everything is fine.